# Joining the Lab 

## Code of Conduct
The Yu Lab is committed to fostering an inclusive, respectful, and professional environment where all members can thrive. Every individual in the lab is expected to treat others with kindness and respect, regardless of their role, background, or identity. Collaboration and open communication are central to our work, and all members are encouraged to share ideas and respectfully challenge one another in the pursuit of scientific excellence.We have a zero-tolerance policy for discrimination, microaggression, or any behavior that undermines the lab’s inclusiveness. 

We uphold the highest standards of integrity in research. All lab members are expected to conduct experiments responsibly, report data honestly, and adhere to ethical guidelines. Data fabrication or manipulation will not be tolerated, as maintaining transparency and accuracy in research is critical to scientific progress.

## Training You Can Expect to Receive

### Chemical Biology

The lab’s focus on developing new chemical tools to probe biological systems, particularly in understanding protein functions and interactions, will provide trainees with a deep understanding of how chemistry can be harnessed to solve biological problems. Beyond basic research, trainees can also expect to apply chemical biology in drug discovery.  

### Bioinformatics

Given the importance of integrating chemistry with computational biology, trainees will receive training in using bioinformatics tools to analyze large omics data. This includes standard software for mass spectrometry data analysis and more advanced data analysis programming language (e.g., R, C#).

### Proteomics

Trainees will gain hands-on experience in mass spectrometry-based proteomics, which is central to the lab’s research. Trainees will also have opportunities to create novel platforms via programming/modifying a million-dollar mass spectrometer.

### Biochemistry and Molecular Biology

Trainees will also engage in foundational training in biochemistry and molecular biology. This includes understanding metabolic pathways, enzyme kinetics, and the molecular mechanisms of gene expression. Such knowledge will enable trainees to appreciate the biochemical context of their research and the interactions between various biomolecules within cellular systems.

## Literature to Warm Up

### Getting started
When just starting off in the lab, I encourage you to read the papers below and to check out resources such as the NCQBCS Mass Spec Summer School videos:  
https://www.ncqbcs.com/resources/training/summer-school/ 
https://www.youtube.com/channel/UC0v4sjdXLMa-OWR7IYeoFoA

### Proteomics Fundamentals
* [The abc's (and xyz's) of peptide sequencing, *Nature Reviews Molecular Cell Biology*, 2004](https://pubmed.ncbi.nlm.nih.gov/15340378/)  
* [An Introduction to Mass Spectrometry-Based Proteomics, *Journal of Proteome Research*,2023](https://pubmed.ncbi.nlm.nih.gov/37260118/)  
* [Single-cell proteomics enabled by next-generation sequencing or mass spectrometry, *Nature Methods*, 2023](https://pubmed.ncbi.nlm.nih.gov/36864196/)

### Multiplexed Quantitative Proteomics
* [Enhanced Multiplexing Technology for Proteomics, *Annual Review of Analytical Chemistry*, 2023](https://www.annualreviews.org/content/journals/10.1146/annurev-anchem-091622-092353)  
* [TMTpro reagents: a set of isobaric labeling mass tags enables simultaneous proteome-wide measurements across 16 samples, *Nature Methods*, 2020](https://www.nature.com/articles/s41592-020-0781-4)
* [Streamlined Tandem Mass Tag (SL-TMT) Protocol: An Efficient Strategy for Quantitative (Phospho)proteome Profiling Using Tandem Mass Tag-Synchronous Precursor Selection-MS3, *Journal of Proteome Research*, 2018](https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00217)

### Intelligent Data Acquisition
* [Full-Featured, Real-Time Database Searching Platform Enables Fast and Accurate Multiplexed Quantitative Proteomics, *Journal of Proteome Research*, 2020](https://pubs.acs.org/doi/10.1021/acs.jproteome.9b00860)
* [Sample multiplexing for targeted pathway proteomics in aging mice, *Proceedings of the National Academy of Sciences of the United States of America*, 2020](https://www.pnas.org/doi/abs/10.1073/pnas.1919410117?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed)
* [Sample multiplexing-based targeted pathway proteomics with real-time analytics reveals the impact of genetic variation on protein expression, *Nature Communications*, 2023](https://www.nature.com/articles/s41467-023-36269-7)

### Data Processing
* [Comet: an open-source MS/MS sequence database search tool, *Proteomics*, 2013](https://pubmed.ncbi.nlm.nih.gov/23148064/)
* [Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry, *Nature Methods*, 2007](https://pubmed.ncbi.nlm.nih.gov/17327847/)  
* [A probability-based approach for high-throughput protein phosphorylation analysis and site localization, *Nature Biotechnology*, 2006](https://pubmed.ncbi.nlm.nih.gov/16964243/)  
* [A Scalable Approach for Protein False Discovery Rate Estimation in Large Proteomic Data Sets, *Molecular Cellular Proteomics*, 2015](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC4563723/)

### Chemoproteomics/Drug Discovery
* [Activity-based protein profiling: from enzyme chemistry to proteomic chemistry, *Annual Review of Biochemistry*, 2008](https://pubmed.ncbi.nlm.nih.gov/18366325/)
* [Quantitative reactivity profiling predicts functional cysteines in proteomes, *Nature*, 2010](https://pubmed.ncbi.nlm.nih.gov/21085121/)
* [Reimagining high-throughput profiling of reactive cysteines for cell-based screening of large electrophile libraries, *Nature Biotechnology*, 2021](https://pubmed.ncbi.nlm.nih.gov/33398154/)
* [Accelerating multiplexed profiling of protein-ligand interactions: High-throughput plate-based reactive cysteine profiling with minimal input, *Cell Chemical Biology*, 2024](https://pubmed.ncbi.nlm.nih.gov/38118439/)

### Proximity Proteomics
* [Creative approaches using proximity labeling to gain new biological insights, *Trends in Biochemical Sciences*, 2024](https://pubmed.ncbi.nlm.nih.gov/38160064/)
* [Proximity Dependent Biotinylation: Key Enzymes and Adaptation to Proteomics Approaches, *Molecular Cellular Proteomics*, 2020](https://pubmed.ncbi.nlm.nih.gov/32127388/) 
* [Spatially resolved proteomic mapping in living cells with the engineered peroxidase APEX2, *Nature Protocol*, 2016](https://pubmed.ncbi.nlm.nih.gov/26866790/)
